htlcswitch/link.go

```diff
@@ -954,6 +964,14 @@ func (l *channelLink) htlcManager() {
 		}
 	}
 
+	// We've successfully reestablished the channel, mark it as such to
+	// allow the switch to forward HTLCs in the outbound direction.
+	l.markReestablished()
+
+	// Now that we've received both funding locked and channel reestablish,
+	// we can go ahead and send the active channel notification.
+	l.cfg.NotifyActiveChannel(*l.ChannelPoint())
```

Comment:
Ah, I see. It is moved here.

Reply:
Could maybe `defer NotifyInactive()` here?

Reply:
good call, w/o doing so it could have caused us to send an inactive ntfn when no corresponding active ntfn had been sent. how that it's deferred, they should be symmetric.